test/suite/wttest.py

```diff
@@ -283,6 +284,7 @@ def extensionsConfig(self):
             filenames = glob.glob(pat)
             if len(filenames) == 0:
                 if skipIfMissing:
+                    self.skipped = True
```

Comment:
If code in other places calls `self.skipTest` (for example `test_bulk01.py`), the `self.skipped` variable won't be set.  We would like to set it in `self.skipTest` itself, but I see that it is defined in a class we don't own.

I think the solution is to create method `WiredTigerTestCase.skipTest(self)`, and set the variable in it, and call `unittest.TestCase.skipTest(self)`.  It's a little more complicated, but I think it's worth getting right for the general case.

Reply:
Thanks for the good suggestion. I've implemented the method solution as suggested in cbc9142.